source/scss/fti-bootstrap/_custom-variables.scss

```diff
@@ -27,6 +27,7 @@
     Text
```

Comment:
we should create a different file for our variables, _custom-variables should be used to override bootstrap variables